docs/developer/testing.md

```diff
@@ -0,0 +1,188 @@
+# Testing guidance
+
+When building Web sites and apps on FAST-DNA it's recommended to perform exhaustive testing to instill the confidence that you've met your customers' expectations.
+
+## Quality checklist
+
+1. Test each page for design and content issues
+    1. Test screen sizes from viewport 1 to 6
+    2. Test screen orientations
+       1. Portrait
+       2. Landscape
+    3. Test word length and case
+        1. Test smallest amount of text
+        2. Test largest amount of text
+        3. Case sensitivity
+2. Test for localization (RTL, LTR)
+    1. English
```

Comment:
I think it is disingenuous to advise testing languages when we don't have infrastructure for doing it - should we remove language testing until we facilitate that with our product?

Reply:
We're sharing what we believe to be important to test. This doesn't mean we have to provide a means of doing all of this.  If you build a website with FAST-DNA then it's up to you to wire up localization not us.  Which is why I removed the localization documentation as well.  But, I think it's useful to share what we believe to be valid testing for quality.

Reply:
That doesn't seem right to me. The context of this markdown file is FAST-DNA. I wouldn't come here expecting to see general guidance of what we advise consumers to do in regards to testing - thats none of our concern. I would expect to come to this file to learn about testing requirements for the FAST-DNA, which currently does not include locale testing.